File: ComercioEletronico/Python/venda.py
class Venda:
    def __init__(self, id, data, carrinho, total, idCliente):
        self.set_id(id)
        self.set_data(data)
        self.set_carrinho(carrinho)
        self.set_total(total)
        self.set_idCliente(idCliente)

    def set_id(self, id):
        self.id = id
    
    def set_data(self, data):
        self.data = data

    def set_carrinho(self, carrinho):
        self.carrinho = carrinho
    
    def set_total(self, total):
        self.total = total
    
    def set_idCliente(self, idCliente):
        self.idCliente = idCliente

    def __str__(self):
        return f"{self.id} - {self.data} - {self.carrinho} - {self.total} - {self.idCliente}"

File: ComercioEletronico/Java/src/Venda.java
import java.time.LocalDate;

public class Venda {
    private int id;
    private LocalDate data;
    private boolean carrinho;
    private float total;
    private int idCliente;

    public Venda(int id){
        setId(id);
    }

    public void setId(int id){
        this.id = id;
    }

    public void setData(LocalDate data){
        this.data = data;
    }

    public void setCarrinho(boolean c){
        this.carrinho = c;
    }

    public void setTotal(float t){
        this.total = t;
    }

    public void setIdCliente(int idCliente){
        this.idCliente = idCliente;
    }

    public int getId(){
        return this.id;
    }

    public LocalDate getData(){
        return this.data;
    }

    public boolean getCarrinho(){
        return this.carrinho;
    }

    public float getTotal(){
        return this.total;
    }

    public int getIdCliente(){
        return this.idCliente;
    }

    public String toString(){
        return String.format("Venda \n ID = %s - Data = %s - Carrinho = %s - Total = %s - ID cliente = %s", id, data, carrinho, total, idCliente);    
    }
}